druid/examples/layout.rs

```diff
@@ -12,31 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! This example shows how to construct a basic layout.
+//! This example shows how to construct a basic layout. using
+//! Collumns and Rows, and loops for repeated Widgets.
 
 use druid::widget::{Button, Flex, Label};
-use druid::{AppLauncher, Color, LocalizedString, Widget, WidgetExt, WindowDesc};
+use druid::{AppLauncher, Color,  Widget, WidgetExt, WindowDesc};
```

Comment:
```suggestion
use druid::{AppLauncher, Color, Widget, WidgetExt, WindowDesc};
```

this was making CI fail; i'll just bunch up these suggestions and merge.